src/main/kotlin/no/nav/syfo/behandlerdialog/api/BehandlerdialogApi.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.syfo.behandlerdialog.api
```

Comment:
Lagde en egen mappe på dette, `behandlerdialog`. Tenkte vi måtte skille det fra `dialogmeldinger`, som jeg anser som stedet for "logikk for å hente fra vår app isdialogmeldinger". Noen som er uenig i det?

Reply:
Høres lurt ut med egen mappe `behandlerdialog` ja 👍 (Det som ligger i dialogmelding-mappen nå er vel prosessering av innkommende meldinger fra padm2, så i hvert fall greit å skille det fra det).